### PR TITLE
Update SSH private key generation in the latest OKD installation documentation

### PR DESCRIPTION
**Pull request by mburke5678**
https://github.com/openshift/okd/issues/445

### CODE REVIEW

**Comment by vrutkovs** [comment]
I think its worth using `ed25519` for OCP as well - its also supported and we should encourage all users to use stronger crypto

**Comment by mburke5678** [comment]
@staebler Can you PTAL to let me know if the installer team approves of this change? Thank you in advance.

**Comment by staebler** [comment]
> @staebler Can you PTAL to let me know if the installer team approves of this change? Thank you in advance.

Looks good to me. Can we make that OCP changes in this PR as well? Or does that need to be a separate PR?

File: modules/ssh-agent-using.adoc [hunk 1]
```diff
@@ -108,54 +97,45 @@ link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html[AWS
...
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+----
+$ ssh-keygen -t ed25519 -N '' \
+    -f <path>/<file_name> <1>
+----
+endif::openshift-origin[]
+
+<1> Specify the path and file name, such as `~/.ssh/id_rsa`, of the SSH key.
+Do not specify an existing SSH key, as it will be overwritten.
```
Comment (staebler) [inline modules/ssh-agent-using.adoc:121]: It took me a couple seconds to digest this advice. Have you seen user run this and accidentally delete ssh keys?

```suggestion
<1> Specify the path and file name, such as `~/.ssh/id_rsa`, of the SSH key to create.
```

**Comment by mburke5678** [comment]
@gpei Can you please review this change to the installer docs?
https://github.com/openshift/openshift-docs/pull/29044/files#diff-af0224102c27ed54d6a7f8ab652cf1b44dd9869b615e1c43d2fe193bc1b77145
https://github.com/openshift/openshift-docs/pull/29044/files#diff-5e1bb338b4de6a5ccc41c16669369a28c98826a77802fd6add6b273e4504fba2

@xiuwang Can you please review this change to the build docs?
https://github.com/openshift/openshift-docs/pull/29044/files#diff-9407125e0d62abb63cc5fb87ebfa69dc67d937ce718d392fed74b2d5159f330b

**Comment by gpei** [comment]
LGTM for the installer part.

**Comment by xiuwang** [comment]
LGTM for the build part.

**Comment by mburke5678** [comment]
@gpei @xiuwang Thank you for your quick review!

**Comment by mburke5678** [comment]
@codyhoag FYI

**Pull request merged**